e2e/web/src/web.test.ts

```diff
@@ -220,10 +232,10 @@ describe('CLI - Environment Variables', () => {
       },
     });
     expect(readFile(`dist/apps/${appName}/main.js`)).toContain(
-      'const envVars = ["test", "52", "QA", "ws-base", "ws-local", "app-base", "app-local", "shared-in-app-local"];'
+      'const envVars = ["test", "52", "QA", "ws-base", "ws-env-local", "ws-local-env", "app-base", "app-env-local", "app-local-env", "shared-in-app-env-local", "shared-in-app-local-env"];'
```

Comment:
Tests are failing here:

The addition of the last `"shared-in-app-local-env"` shouldn't be there because it should be overwritten.